docs/collecting-container-logs.md

```diff
@@ -205,6 +207,40 @@ sumologic:
               key: redundant-attribute
 ```
 
+#### Adding custom fields
+
+To add a custom [field][sumo_fields] named `static-field` with value `hardcoded-value` to logs, use the following configuration:
+
+```yaml
+sumologic:
+  logs:
+    container:
+      otelcol:
+        extraProcessors:
+        - resource/add-static-field:
+            attributes:
+            - action: insert
+              key: static-field
+              value: hardcoded-value
+```
+
+To add a custom field named `k8s_app` with a value that comes from e.g. the pod label `app.kubernetes.io/name`, use the following configuration:
+
+```yaml
+sumologic:
+  logs:
+    container:
+      otelcol:
+        extraProcessors:
+        - resource/add-k8s_app-field:
+            attributes:
+            - action: insert
+              key: k8s_app
+              from_attribute: pod_labels_app.kubernetes.io/name
+```
+
+**NOTE** Make sure the field is [added in Sumo Logic][sumo_add_fields].
```

Comment:
Will that work if we add it to our configuration for field creation?

Reply:
Yes it will, you just need to duplicate the whole list of fields...

```yaml
sumologic:
  logs:
    fields:
    - cluster
    - container
    - daemonset
    - deployment
    - host
    - namespace
    - node
    - pod
    - service
    - statefulset
    - k8s_app
```

Reply:
I would consider adding additional fields in the future 🤔

Reply:
There's a chance we won't have to add the basic k8s metadata fields at all, in which case the problem will solve itself to an extent.